src/pages/reference/transforms/naming-convention.md

```diff
@@ -48,6 +48,21 @@ For information about `bare` and `wrap` modes, please read the [dedicated sectio
 
 ## Config API Reference
 
+<InlineAlert variant="info" slots="text"/>
+
+<p>
+The following case options should not be used because they violate the <a href="https://spec.graphql.org/June2018/#sec-Names)">GraphQL spec</a>
```

Comment:
Sorry @jhadobe, I linked an older spec on the ticket. Can we use this one instead? 
https://spec.graphql.org/October2021/#sec-Names